charts/grafana/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: grafana
-version: 6.57.4
-appVersion: 9.5.5
+version: 9.6.0
+appVersion: 10.0.1
```

Comment:
@zzzinho 
Please bump to:

```suggestion
version: 6.58.0
appVersion: 10.0.1
```

@rverenich is not one of the chart maintainers. You should not take his advice.

Reply:
I committed suggestion :)

Reply:
Hey, my advice was not to go with 7.0.0 chart version. Just bump 6.x chart version with `appVersion: 10.0.1`
@zzzinho , please approve and let's go further:) Thanks in advance